pkgs/applications/graphics/awesomebump/default.nix

```diff
@@ -2,12 +2,12 @@
 
 
 let
-  version = "5.1";
+  version = "5.1.1";
 
   src = fetchgit {
     url = "https://github.com/kmkolasinski/AwesomeBump.git";
-    rev = "Winx32v${version}";
-    sha256 = "1c8b9jki0v8kzkvsvyv7q1w3s7j40br6ph15hh2xi0a1mpwckq56";
+    rev = "Linuxv${version}";
```

Comment:
That tag name is really misleading, unfortunately (the Windows and Linux releases are tagged at the same commit). Nothing we can do about that but it threw me for a loop.